lib/semantics/symbol.h

```diff
@@ -679,4 +679,4 @@ inline bool ProcEntityDetails::HasExplicitInterface() const {
   return false;
 }
 }
-#endif  // FORTRAN_SEMANTICS_SYMBOL_H_
+#endif  // FORTRAN_SEMANTICS_SYMBOL_H_
```

Comment:
It looks like you accidentally remove the newline from the end of this file.

Reply:
Oops. VS Code has been doing really weird things to this file specifically.